config/prometheus/monitor.yaml

```diff
@@ -5,8 +5,8 @@ kind: ServiceMonitor
 metadata:
   labels:
     control-plane: controller-manager
-  name: controller-manager-metrics-monitor
-  namespace: system
+  name: lvm-operator-controller-manager-metrics-monitor
```

Comment:
any reason for adding the namespace as prefix? if this isn't mandatory for the metrics, can we please not use this?

Reply:
This prefix makes it more readable. In fact, we should add this prefix to all the lvm-operator objects.

Reply:
Yes, you should add prefix to all your resources.
Use https://github.com/red-hat-storage/lvm-operator/blob/main/config/default/kustomization.yaml#L4-L9 for prefix.
Also, try adding labels via https://github.com/red-hat-storage/lvm-operator/blob/main/config/default/kustomization.yaml#L11-L13 for easy queries later on.

Reply:
- this prefix usage was discussed earlier, we decided not to use this and follow ocs-operator convention.
- the reason was, it's difficult to get this prefix in code where we are associating serviceaccount names with deployments

Reply:
We can modify the kustomize files to include the prefix explicitly. I agree that the prefix should be added.

Reply:
ack, then we need to settle on a consistent name and use that in code as well while supplying serviceaccount names